include/deal.II/hp/dof_handler.h

```diff
@@ -122,15 +124,9 @@ namespace hp
    * circumstances. In particular, the following rules apply:
    * - Upon mesh refinement, child cells inherit the active FE index of
    *   the parent.
-   * - On the other hand, when coarsening cells, the (now active)
-   *   parent cell will not have an active FE index set and you will
-   *   have to set it explicitly before calling
-   *   hp::DoFHandler::distribute_dofs(). In particular, to avoid
-   *   stale information to be used by accident, this class deletes
-   *   the active FE index of cells that are refined after inheriting
-   *   this index to the children; this implies that if the children
-   *   are coarsened away, the old value is no longer available on the
-   *   parent cell.
+   * - When coarsening cells, the (now active) parent cell will be assigned
+   *   an active FE index that is determined from its (no longer active)
+   *   children, following the FiniteElementDomination logic.
```

Comment:
can you be more specific? namely say that you are choosing the least dominant element among the elements that dominate all of the elements used on the children (and possibly a reference to the function in FECollection you are calling because the concept isn't all that easy to understand :-)